pasqal_cloud/client.py

```diff
@@ -206,6 +209,51 @@ def rebatch(
         )["data"]
         return new_batch
 
+    def get_jobs(
```

Comment:
It looks like the other methods of this client are prefixed by underscores to discourage their direct usage by the users.
Maybe you should do the same ?

Reply:
well I followed the same pattern as rebatch above, that is not a private method. I can change it and rebatch to a private method maybe ? To have something more consistent